packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Open the Sign-Out Page
+---
+To sign out users, open a web view and display the hosted sign-out page.
+
+<StackSelector snippet="signout"/>
+
+Signing out signs the user out if the subject in the ID token matches the current Okta session. A `post_logout_redirect_uri` may be specified to redirect the browser after the sign out is performed, for example, `com.oktaorg.name:/logout`. Otherwise, the browser is redirected to the Okta sign-in page.
```

Comment:
One small change with this first sentence:

Users are signed out if the subject in the ID token matches the current Okta session.